src/components/banner/index.tsx

```diff
@@ -21,15 +21,17 @@ const emptyElement = {
 export function IterableEmbeddedBanner({
   appPackageName,
   message,
-  parent = emptyElement,
-  img = emptyElement,
-  title = emptyElement,
-  primaryButton = emptyElement,
-  secondaryButton = emptyElement,
-  body = emptyElement,
-  buttonsDiv = emptyElement,
-  textTitle = emptyElement,
-  textTitleImg = emptyElement,
+  elements = {
```

Comment:
Just a thought, but hopefully this `elements` property doesn't get muddled/confused down the line with `message.elements` since both appear near each other, I wonder if maybe there's another name we could give this "elements" that something to do with the fact that we're pulling styles from them 🤔

Reply:
hmmmm good callout... will think on this some before merging.